.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+# Related Issue
+- issue goes here
```

Comment:
```suggestion
Fixes #00 <!-- Replace `00` with the Issue Number -->
```

https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue This helps in linking the issue in a better way.

Reply:
I had committed your suggestion